connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java

```diff
@@ -236,20 +239,28 @@ protected void poll(long timeoutMs) {
         }
     }
 
+    private void doCommitSync(Map<TopicPartition, OffsetAndMetadata> offsets, int seqno) {
+        try {
+            consumer.commitSync(offsets);
+            lastCommittedOffsets = offsets;
+            onCommitCompleted(null, seqno);
+        } catch (WakeupException e) {
+            // retry the commit to ensure offsets get pushed, then propagate the wakeup up to poll
+            doCommitSync(offsets, seqno);
+            throw e;
```

Comment:
@hachikuji  Why don't we catch the exception here?  If the task is being stopped, a wakeupexption is kind of noise in log especially for the application which need to monitor error log.